### ISSUE TITLE
"error: SEH '__try' is not supported on this target" Clang Windows ARMv7 cross-compile fails

### ISSUE THREAD

**Issue by darktohka** [kcat/openal-soft#742]
I am trying to cross-compile OpenAL Soft to Windows, specifically the ARMv7 architecture, with Clang 14.0.6.

When compiling OpenAL Soft master with Clang 14.0.6 and CMake 3.23.2, on a fully-upgraded Arch Linux x86_64 host, using the Windows MSVC toolchain, the following error happens:

```
/home/user/Desktop/build-tc/openal/common/threads.cpp:52:5: error: SEH '__try' is not supported on this target
````

The following function is broken: https://github.com/kcat/openal-soft/blob/master/common/threads.cpp#L35

Unfortunately, `__try` is not available on ARMv7. AArch64/ARM64 is not affected. Perhaps the thread rename could be disabled when building OpenAL Soft in Release mode?

**Comment by kcat** [comment]
It's still useful even in Release mode, but this should be fixed now with commit 7dcd2a86b8ff6c4f1a2872cea626d0d7a4adc269 which skips it for ARMv7 targets.

**Comment by darktohka** [comment]
Thank you so much kcat! ARMv7 build works fine now.
